gorp.go

```diff
@@ -257,15 +257,15 @@ func columnToFieldIndex(m *DbMap, t reflect.Type, name string, cols []string) ([
 			cArguments := strings.Split(field.Tag.Get("db"), ",")
 			fieldName = cArguments[0]
 
+			if fieldName == "" || fieldName == "-" {
```

Comment:
With respect to the partial revert, can you elaborate on the impact to the bot fields in question?

Reply:
Should be none and we recover the Rename feature

Reply:
This should be fine. We were not using the rename feature at the time so I didn't worry about breaking it. It keeps the change acutally needed from the other PR.